sklearn/utils/tests/test_tags.py

```diff
@@ -1,47 +1,26 @@
 import pytest
 
-from sklearn.base import BaseEstimator
-from sklearn.utils._tags import (
-    _DEFAULT_TAGS,
-    _safe_tags,
-)
+from sklearn.base import BaseEstimator, RegressorMixin, TransformerMixin
+from sklearn.utils._tags import get_tags
 
 
 class NoTagsEstimator:
     pass
 
 
-class MoreTagsEstimator:
-    def _more_tags(self):
-        return {"allow_nan": True}
-
-
-@pytest.mark.parametrize(
-    "estimator, err_msg",
-    [
-        (BaseEstimator(), "The key xxx is not defined in _get_tags"),
-        (NoTagsEstimator(), "The key xxx is not defined in _DEFAULT_TAGS"),
-    ],
-)
-def test_safe_tags_error(estimator, err_msg):
-    # Check that safe_tags raises error in ambiguous case.
-    with pytest.raises(ValueError, match=err_msg):
-        _safe_tags(estimator, key="xxx")
+class ClassifierEstimator:
```

Comment:
How come ClassifierMixin doesn't work?

Reply:
It might, I guess the idea is here to test cases where we haven't inherited from those mixins.

Reply:
> It might, I guess the idea is here to test cases where we haven't inherited from those mixins.

Can you add this comment in the code?